### DISCUSSION TITLE
Not able to connect to milvus cluster on AKS

### DISCUSSION THREAD

**Discussion by manas007** [milvus-io/milvus#31722] [Q&A and General discussion]
Followed the deployment guide here: https://milvus.io/docs/azure.md

kubectl get services 
Output:
<img width="328" alt="image" src="https://github.com/milvus-io/milvus/assets/19229097/744be3a1-1e36-4d45-a97b-c08055908151">

kubectl get pods -o wide
Output:

<img width="526" alt="image" src="https://github.com/milvus-io/milvus/assets/19229097/759c5a78-8dd8-4562-b808-6966caff7468">

Values.yaml 
![image](https://github.com/milvus-io/milvus/assets/19229097/a737cd71-d39e-4f4d-ada7-c7111fa43741)

when connecting to milvus using connection.connect and updated host and port number, running into the error
<MilvusException: (code=2, message=Fail connecting to server on XXXX. Timeout)>

Can someone guide ?

**Comment by yhmo** [comment]
Wait @LoveEachDay to help.

**Reply by LoveEachDay** [reply]
@manas007 Could you provide the object storage config? Guess milvus cannot connect the object storage service.
Have you setup the following items in your `values.yaml`:

<img width="704" alt="image" src="https://github.com/milvus-io/milvus/assets/1573213/9774dd67-791f-4f63-8c11-22b7f3b3ddf7">

**Reply by manas007** [reply]
Hi @LoveEachDay : thanks for answering, yes. I had the values.yaml file setup as described in the documentation. 

Here's the storage service config in my values.yaml

<img width="634" alt="Screenshot 2024-03-29 at 10 47 42 AM" src="https://github.com/milvus-io/milvus/assets/19229097/af6c7cca-70b1-4ac2-968f-8089e56f5e94">

**Comment by manas007** [comment]
Any advise @LoveEachDay  ?
